addOns/openapi/src/test/java/org/zaproxy/zap/extension/openapi/v3/BodyGeneratorUnitTest.java

```diff
@@ -301,6 +301,79 @@ public void shouldReadAdditionalMapObject() throws IOException {
                 request);
     }
 
+    @Test
+    public void shouldUseExampleFromRequest() throws IOException {
```

Comment:
`FromSchema` or `InSchema`?

Reply:
Did you mean `Request` instead of `Schema`?
I've renamed the Method to `...InRequest` which is more fitting 👍

Reply:
I thought that was referring to where the examples were (following tests are named like that), not where they will be used(?). Technically they are all in the request so seems redundant.

Reply:
Oh yeah that a good point. Removed the part.